pages/landing.tsx

```diff
@@ -0,0 +1,133 @@
+import Particles from "react-tsparticles";
+import ContinueButton from "../components/ContinueButton";
+
+const Welcome = () => (
```

Comment:
This is the previous `welcome.tsx`, just moved to a new home at `landing.tsx` until we decide if/where to use it